src/index.html

```diff
@@ -7,8 +7,67 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="is-active link">
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">
+              Tablets
```

Comment:
Please provide spaces between adjacent elementі
```suggestion
              Apple
            </a>
          </li>
          
          <li class="nav-item">
            <a href="#" class="link">
              Samsung
            </a>
          </li>
          
          <li class="nav-item">
            <a href="#" class="link">
              Smartphones
            </a>
          </li>
          
          <li class="nav-item">
            <a
              href="#"
              class="link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          
          <li class="nav-item">
            <a href="#" class="link">
              Gadgets
            </a>
          </li>
          
          <li class="nav-item">
            <a href="#" class="link">
              Tablets
```